src/test/kotlin/uk/gov/justice/digital/hmpps/electronicmonitoringdatastoreapi/resources/OrderControllerTest.kt

```diff
@@ -9,36 +8,41 @@ import org.mockito.Mockito
 import org.mockito.Mockito.mock
 import org.mockito.Mockito.times
 import org.mockito.Mockito.`when`
+import org.springframework.boot.test.autoconfigure.json.JsonTest
 import org.springframework.http.ResponseEntity
-import org.springframework.security.access.AccessDeniedException
+import org.springframework.security.core.Authentication
 import org.springframework.test.context.ActiveProfiles
 import uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.model.AthenaQueryResponse
 import uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.model.KeyOrderInformation
 import uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.model.OrderInformation
 import uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.repository.OrderInformationRepository
+import uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.resource.OrderController
+import uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.service.internal.AuditService
 
 @ActiveProfiles("test")
+@JsonTest
 class OrderControllerTest {
-
-  private val objectMapper = ObjectMapper()
-  private lateinit var repository: OrderInformationRepository
-  private lateinit var sut: OrderController
+  private val repository: OrderInformationRepository = mock()
```

Comment:
I'd have thought we wanted this in the Setup() method, so we can validate how many times things are called in each test?

If it's fine here/that still works I'm fine with that, but unsure why you moved it.